src/middlewared/middlewared/plugins/cloud_backup/sync.py

```diff
@@ -54,8 +55,10 @@ async def restic_backup(middleware, job, cloud_backup, dry_run):
             cmd = ["--stdin", "--stdin-filename", "volume"]
         else:
             await check_local_path(middleware, local_path)
+            if cloud_backup["snapshot"]:
```

Comment:
Why no snapshots for zvol?

Reply:
We already have different snapshot logic for zvols that I didn't want to mess with, see line 26.